packages/hash/frontend/src/graphql/queries/ontology/entity-type.queries.ts

```diff
@@ -20,6 +20,47 @@ export const getAllLatestEntityTypesQuery = gql`
   }
 `;
 
+export const getEntityTypeRootedSubgraphQuery = gql`
+  query getEntityTypeRootedSubgraph(
+    $entityTypeId: String!
+    $referencedDataTypesDepth: Int
+    $referencedPropertyTypesDepth: Int
+    $referencedLinkTypesDepth: Int
+    $referencedEntityTypesDepth: Int
+  ) {
+    getEntityType(entityTypeId: $entityTypeId) {
+      entityTypeId
+      ownedById
+      accountId
+      entityType
+      referencedDataTypes(depth: $referencedDataTypesDepth) {
+        dataTypeId
+        ownedById
+        accountId
```

Comment:
Might have missed this somewhere, but how come we have the `accountId` in this query well when we control the ontology types?

Reply:
Good point, this can be removed I think but will have to be done in follow-up. Will do a drive-by in an upcoming PR